### PR TITLE
Support ramsey/uuid version 4

### PR DESCRIPTION
**Pull request by tsctao**

### CODE REVIEW

**Comment by TomasVotruba** [comment]
Better use 4.0 to keep consistent version

**Comment by tsctao** [comment]
> Better use 4.0 to keep consistent version format

I followed [ramsey/uuid's documentation](https://uuid.ramsey.dev/en/latest/upgrading/3-to-4.html).

If we force version 4, it will conflict with user's projects that are not ready to upgrade ramsey/uuid to version 4?

File: composer.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
         "symfony/string": "^5.0",
         "symplify/package-builder": "^7.2",
         "nette/utils": "^3.0",
-        "ramsey/uuid": "^3.9"
+        "ramsey/uuid": "^3.9|^4"
```
Comment (Kocal) [inline composer.json]:
```suggestion
        "ramsey/uuid": "^3.9|^4.0"
```

**Comment by tsctao** [comment]
Oh, I misunderstood. Thank you.

**Comment by TomasVotruba** [comment]
@tsctao I was on mobile so I didn't realize I could use suggestion. Code is always better than words.

Thank you both!

**Pull request merged**